src/bp/ui-studio/src/web/components/Layout/BottomPanel/Debugger/index.tsx

```diff
@@ -198,17 +198,13 @@ export class Debugger extends React.Component<Props, State> {
   }
 
   render() {
-    if (this.props.hidden) {
-      return null
-    }
-
     const hasEvent = !!this.state.event
     const ndu = _.get(this.state, 'event.ndu')
 
     return (
       <Tabs
         id="tabs"
-        className={cx(btStyle.tabs)}
+        className={cx(btStyle.tabs, this.props.hidden && btStyle.hidden)}
```

Comment:
The classnames lib has a special notation for optional classes, which I find easier to read: 
`cx(btStyle.tabs, { [btStyle.hidden]: this.props.hidden)`

Reply:
Sweet, thanks for the tip